src/components/NewLocationPage/CompareLocationTabs/CompareLocationTabs.style.tsx

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+import { COLOR_MAP } from 'common/colors';
+import ToggleButton from '@material-ui/lab/ToggleButton';
+import ToggleButtonGroup from '@material-ui/lab/ToggleButtonGroup';
+
+export const ButtonGroup = styled(ToggleButtonGroup)`
+  background-color: ${COLOR_MAP.GREY_1};
+  border: 1px solid ${COLOR_MAP.GREY_2};
+  border-radius: 4px;
+`;
+
+export const Button = styled(ToggleButton)`
+  padding: 0.63rem;
+  border: none;
+  .MuiToggleButton-label {
+    color: ${COLOR_MAP.GREY_3};
+    text-transform: none;
+    line-height: 1rem;
+  }
+  &.Mui-selected {
+    background-color: white;
+    .MuiToggleButton-label {
+      color: ${COLOR_MAP.BLACK};
+    }
+    font-weight: 500;
```

Comment:
`${props => props.theme.fonts.regularBookMidWeight}` rather than hard coding the font weight

Reply:
also- re: the mocks, it looks like the increased font weight should only apply to the selected button (its currently applying to all of them)

Reply:
so i think if you move the midWeight style up a line (into the selected styles), you should be good to go

Reply:
👍🏼